app/models/vim_performance_trend.rb

```diff
@@ -113,7 +113,7 @@ def self.build(perfs, options)
       row[:count_of_trend] = trend_data[name][:count]
 
       # trend direction
-      row[:direction_of_trend] = row[:slope] ? row[:slope] > 0 ? "Up" : row[:slope] < 0 ? "Down" : "Flat" : nil
+      row[:direction_of_trend] = (row[:slope] > 0 ? "Up" : row[:slope] < 0 ? "Down" : "Flat") if row[:slope]
```

Comment:
This whole line should be rewritten to not use nested ternaries at all, since they are frowned upon.

Reply:
`if` FTW